changelog/fragments/replace-max-workers.yaml

```diff
@@ -0,0 +1,4 @@
+entries:
+  - description: >
+      Add "--max-concurrent-reconciles" flag to helm binary.
```

Comment:
```ymal
entries:
  - description: >
       **(Valid just for Ansible/Helm-based Operators)**. The flag `max-workers` is no longer supported. Use `--max-concurrent-reconciles` instead of that. 
    
    kind: "removal"
    breaking: true
    
    migration:
      header:  The flag `max-workers` is no longer supported.
      body: >
       TBD
```

Reply:
We need mark as a breaking change we are removing a flag. The TBD is because we agreed in to do that since we will prepare a doc for 1.0.0

Reply:
This should probably be `kind: change` and `breaking: true`

We can fill in the migration for this since it's pretty straightforward (i.e. switch to "--max-concurrent-reconciles")